docs/en/siem/field-ref.asciidoc

```diff
@@ -2,15 +2,17 @@
 [chapter, role="xpack"]
 = Field reference and mappings
 
-This section lists all fields required to populate {siem-soln} data.
+This section lists the fields required to populate all {siem-soln} data.
 
-NOTE: Unless noted otherwise, all listed fields are {ecs-ref}[ECS fields]. 
+NOTE: Unless noted otherwise, all listed fields are {ecs-ref}[ECS fields].
+Whenever possible, it is recommended to use ECS fields. In future releases,
+{siem-soln} will adopt more ECS fields and non-ECS fields will be deprecated. To reduce time spent on updating you ingestion pipelines in the future, see {ecs-ref}/ecs-converting.html[how to convert data to ECS].
 
 [float]
 [[siem-global-fields]]
 == Global fields
 
-Fields required for populating all SIEM data:
+Fields required for populating SIEM data:
```

Comment:
same comment as above about "required"

Reply:
I think these two fields are essential for getting meaningful data into SIEM.